internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -1263,7 +1275,14 @@ func CordonAndDrainNode(seq runtime.Sequence, data interface{}) (runtime.TaskExe
 
 		defer kubeHelper.Close() //nolint:errcheck
 
-		return kubeHelper.CordonAndDrain(ctx, nodename)
+		err = kubeHelper.CordonAndDrain(ctx, nodename)
+		if _, ok := data.(runtime.ResetOptions); ok {
+			if apierrors.IsTimeout(err) || apierrors.IsServerTimeout(err) {
```

Comment:
I guess this is not exhaustive list of errors, it might e.g. "connection refused" easily? 

should we in general report, but ignore errors here?